### DISCUSSION TITLE
Using GitHub gist to post as a bot instead of me

### DISCUSSION THREAD

**Discussion by ioistired** [community/community#21881] [New to GitHub]
<p>I currently run a Discord bot that used to use anonymous GitHub Gists. Now that anonymous Gists have been disabled :angry: I need another solution.<br>
Since the lists use inline images and tables, there’s not so many alternatives.</p>
<p>From the API docs:<br>
“You can read public gists anonymously, but you must be signed into GitHub to create gists. To read or write gists <em>on a user’s behalf</em>, you need the gist <a href="https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/" rel="nofollow noopener">OAuth scope</a> and a token.” (emphasis mine)</p>
<p>Is there a way to have the author show up as created by the bot, instead of created by me? Do I have to create a new GitHub account for this? Should I make an organization account?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you want the Gist to show up as created by a different account than yours, you’ll need to create a new machine account for that. An organization can’t log in as a user, so it wouldn’t work for what you’re describing.</p>

**Comment by ioistired** [comment]
<p>Is a “machine account” a specific thing, or just a user account that happens to only be used by a machine?</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/bmintz">@bmintz</a> It’s a regular user account used by a bot/machine.</p>
